RDS/layer3_central_services/token_storage/src/lib/Storage.py

```diff
@@ -451,12 +451,9 @@ def internal_removeToken(self, user: User, token: Token):
             for index, crnt in enumerate(self._storage[user.username]["tokens"]):
                 if token.servicename == crnt.servicename:
                     logger.debug("found")
```

Comment:
I know this is strictly speaking not part of the PR, but as you're in that area already, could you change this debug log to something more meaningful?